cmd/service.go

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Do we still need these license text fields in every file? Managing the one LICENSE.md file in the main directory is preferable, because we might forget to do this for newer files. Certainly nothing that needs to be addressed in this PR, but I did want to point it out before forgetting.

Reply:
Yeah, the norm is to have a license header in each file and the full license text in `LICENSE.md`. I have my IDE configured to automatically add this to new files, which we can look into enforcing in CI down the road.

Reply:
The [Apache website](https://infra.apache.org/apply-license.html#copy-per-file) suggests it may not be needed for every file.

Reply:
Sweet, I'm down with removing them to clean things up a bit. I'll remove the new ones from this PR then circle back to remove them from the rest of the repo.